unfoldingWord/translation-helps-rcl
# Feature jincy 457

## Describe what your pull request addresses

- Addresses this(https://app.zenhub.com/workspaces/gatewayedit-5fa302df5c4ec9001f1d5d54/issues/gh/unfoldingword/gateway-edit/457) issue

## Test Instructions

- In merge dialoge component you can see two examples of cards which are ready to merge as a user you can select or deselect the cards
